log.txt

```diff
@@ -0,0 +1,1330 @@
+spawn ./run.sh

```

Comment:
Czy to jest wygenerowany plik? Jeśli tak - po co go wersjonujemy?

Reply:
Dodałem do pliku .gitignore.

Reply:
Pamiętaj, by także usunąć z wersjonowania. Samo ignorowanie, jeśli plik jest wersjonowany, nie wystarcza.